stripe-elements.html

```diff
@@ -477,6 +524,31 @@
       }
     }
 
+    /**
+     * Checks if the Stripe form is valid.
+     * @return {Boolean} true if the Stripe form is valid
+     */
+    validate() {
+      let isValid = this.isComplete && !this.isEmpty && !this.hasError;
+      if (!isValid) {
+        let error = {
+          message: "Credit card information is invalid.",
+        };
+        this._setError(error);
+      }
+      return isValid;
+    }
+
+    /**
+     * Checks for potential validity. A potentially valid form is one that
+     * is not empty, not complete and has no error. A validated form also counts
+     * as potentially valid.
+     * @return {Boolean} true if the Stripe form is potentially valid
+     */
+    isPotentiallyValid() {
```

Comment:
Interesting... What's the use case?

Reply:
It's something Square offers on their payment form. For me personally, I won't use it but it was easy to add in while I put in the rest of the methods so I thought it didn't hurt to add some additional functionality. I guess a dev could show some styling once the form isn't empty and is potentially a valid input.